kube/pod.go

```diff
@@ -272,31 +269,17 @@ func getContainerLivenessProbe(role *model.Role) (helm.Node, error) {
 		return nil, nil
 	}
 
-	var probe *helm.Mapping
+	var err error
+	probe := helm.NewMapping()
 	if role.Run.HealthCheck != nil && role.Run.HealthCheck.Liveness != nil {
-		var complete bool
-		var err error
-		probe, complete, err = configureContainerProbe(role, "liveness", role.Run.HealthCheck.Liveness)
+		probe, _, err = configureContainerProbe(role, "liveness", role.Run.HealthCheck.Liveness)
 
 		if probe.Get("initialDelaySeconds").String() == "0" {
 			probe.Add("initialDelaySeconds", defaultInitialDelaySeconds)
 		}
-		if complete || err != nil {
-			return probe, err
-		}
-	}
```

Comment:
I think this check for `complete` should remain, so that we don't emit an empty liveness probe into the YAML.